pkg/apis/provisioning/v1alpha1/provisioner.go

```diff
@@ -15,13 +15,36 @@ limitations under the License.
 package v1alpha1
 
 import (
+	f "github.com/awslabs/karpenter/pkg/utils/functional"
 	v1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+// ProvisionerSpec is the top level provisioner specification.
```

Comment:
It would be great to add a couple more clarifications (not need to belabor, just a few words) - can you have just one `Provisioner` in a cluster even if you have multiple architectures? Also, *can* you have multiple `Provisioner`s if you have some reason to? (Or would the latter just never be a Thing?)